vignettes/introduction.Rmd

```diff
@@ -0,0 +1,135 @@
+---
+title: "An Introduction to sgdnet"
+author: "Johan Larsson"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+bibliography: "sgdnet.bib"
+vignette: >
+  %\VignetteIndexEntry{An Introduction to sgdnet}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Background and motivation
+
+The goal of **sgdnet** is to be
+a one-stop solution for fitting elastic net-penalized generalized linear 
+models. This is of course not novel in and by itself. Several packages,
```

Comment:
I think this is too broad - I think it's unlikely we'll outperform `glmnet` in the `p \gg n` setting or the `p \approx n \gg 1` setting. I'd focus our attention on the `n \gg p` setting.